### PR TITLE
Update aws-nitro-enclaves-cli to v1.0.12 to fix nitro-base build

### PR DESCRIPTION
**Pull request by mathias-arm**
The compilation failure in `nitro-base` is due to an update to the zeroize needed by cargo-audit

### CODE REVIEW

**Review by dreemkiller — DISMISSED** [review]

**Comment by dominic-mulligan-arm** [comment]
I've managed to build the image using these changes.  I'll approve, though we should wait for @dreemkiller's approval before merging.

**Review by dominic-mulligan-arm — DISMISSED** [review]

**Comment by mathias-arm** [comment]
Merged both commits and signed it.

**Review by dominic-mulligan-arm — APPROVED** [review]

**Review by dreemkiller — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
+1+1 = +1 reached, merging.

**Pull request merged**
